pubspec.yaml

```diff
@@ -8,10 +8,10 @@ author: Devon Carew <devoncarew@google.com>
 environment:
   sdk: '>=1.10.0 <2.0.0'
 dependencies:
-  args: '^0.13.1'
-  logging: '^0.11.0'
+  args: '^0.13.2'
+  logging: '^0.11.1'
```

Comment:
If it doesn't depend on strictly greater than 0.11.1, let's stick with the 0.11.0 constraint (same with args). That'll reduce the number of downstream version conflicts.